web/controllers/movie_round_controller.ex

```diff
@@ -10,4 +10,23 @@ defmodule MovieWagerApi.MovieRoundController do
 
     json(conn, serialized_rounds)
   end
+
+  def create(conn,%{"data" =>  %{"attributes" => movie_round_params}}) do
+    changeset = MovieRound.changeset(%MovieRound{}, movie_round_params)
+
+    case Repo.insert(changeset) do
```

Comment:
This is just the `case/switch` for Elixir.  One of the big things with elixir is pattern matching for params.  So, rather having to write `case x === 'foo'` we can simply write what the result of the case is, in this case a tuple of `{:ok, movieRound}` or `{:error, changeset}`.